### DISCUSSION TITLE
Obstacle avoidance in MPC of a drone in simulation

### DISCUSSION THREAD

**Discussion by uzakotim** [ctu-mrs/mrs_uav_system#138] [Q&A] [answered]
Hello 😀 Do you know how to turn off obstacle avoidance of a drone in simulation or to reduce prediction horizon in MPC?
Thank you,
Timur

**Comment by matemat13** [comment, marked as answer]
Hello Timur!
Surprisingly, we do ;)
The best way to do it is [using custom configs](https://ctu-mrs.github.io/docs/system/custom_configs.html) and change the [`collision_avoidance/enabled` parameter of `mrs_uav_trackers`](https://github.com/ctu-mrs/mrs_uav_trackers/blob/master/config/default/mpc_tracker.yaml#L23) to `false`. You can change the prediction horizon using the same method.

*Note:* It's not recommended to change the parameter directly in the `mrs_uav_trackers/config/default/mpc_tracker.yaml` file as that will likely be overriden by a `git pull` or a similar command and is polluting the repository with non-general changes. Use the custom configs method!

**Reply by uzakotim** [reply]
Thank you!
